ClientApp/src/Pages/WhaleSpecies/WhaleSpecies.tsx

```diff
@@ -1,12 +1,53 @@
 import "./WhaleSpecies.scss";
```

Comment:
there's nothing at the moment that actually allows us to change what the selected image at the top of this page I think which will need to be fixed :)

Reply:
After having a little look it looks like you can kind of click/press right/left to navigate between the whales but it seems very buggy to me from a quick play - is it working exactly as intended for you locally?